cura/CrashHandler.py

```diff
@@ -194,6 +196,17 @@ def _informationWidget(self):
         self.data["os"] = {"type": platform.system(), "version": platform.version()}
         self.data["qt_version"] = QT_VERSION_STR
         self.data["pyqt_version"] = PYQT_VERSION_STR
+        self.data["locale_os"] = locale.getlocale(locale.LC_MESSAGES)[0] if hasattr(locale, 'LC_MESSAGES') else locale.getdefaultlocale()[0]
+        self.data["locale_cura"] = self.cura_locale
+
+        with configure_scope() as scope:
+            scope.set_tag("qt_version", QT_VERSION_STR)
+            scope.set_tag("pyqt_version", PYQT_VERSION_STR)
+            scope.set_tag("os", platform.system())
+            scope.set_tag("os_version", platform.version())
+            scope.set_tag("locale_os", self.data["locale_os"])
+            scope.set_tag("locale_cura", self.cura_locale)
+            scope.set_tag("is_enterprise", ApplicationMetadata.IsEnterpriseVersion)
```

Comment:
Currently in our crash report screen we display the following information only:

* Cura version
* Cura build type
* Platform / OS
* Qt Version
* PyQt Version
* OpenGL version

Should we add the OS locale, Cura locale and isEnterprise to this? Perhaps enterprise or not can be displayed together with the build type?

Reply:
I'm also kinda missing buildtype still, but it's okay to add that when we start using it from something other than enterprise. development builds